common/app/common/TrailsToRss.scala

```diff
@@ -86,7 +86,7 @@ object TrailsToRss extends implicits.Collections {
         val imageMetadata = new Metadata()
         trailAsset.caption.foreach({ d => imageMetadata.setDescription(stripInvalidXMLCharacters(d)) })
         trailAsset.credit.foreach { creditName =>
-          val credit = new Credit(null, null, creditName)
+          val credit = new Credit(null, null, stripInvalidXMLCharacters(creditName))
```

Comment:
As discussed in person, it would be great to add a comment explaining what this is actually doing https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/common/app/common/TrailsToRss.scala#L23